src/Dapper.FluentMap.Dommel/Mapping/DommelEntityMap.cs

```diff
@@ -44,5 +44,15 @@ protected void ToTable(string tableName)
         {
             TableName = tableName;
         }
+
+        /// <summary>
+        /// Sets the table name for the current entity.
+        /// </summary>
+        /// <param name="tableName">The name of the table in the database.</param>
+        /// <param name="schemaName">The name of the table schema in the database.</param>
+        protected void ToTable(string tableName, string schemaName)
```

Comment:
Would it make sense to put the schema as the first parameter? Since it's the first element in the table name as well.